README.md

```diff
@@ -68,6 +74,7 @@ Steps (assumes a linux system)
 
   `http://localhost:8000/approval_polls`
 
+  (However, if you want Django to run on a public IP, make sure you update the ALLOWED_HOSTS variable to include the required IP address in your local_settings.py.)
```

Comment:
Sounds great. Could you also include a link to the [ALLOWED_HOSTS documentation] (https://docs.djangoproject.com/en/1.8/ref/settings/#allowed-hosts) and add a cautionary note that 'localhost' should not be used in production? Thanks!